### PR TITLE
Updated collection-index.yml

### PR DESCRIPTION
**Pull request by ChaosWars**
Added Synology features

## What type of PR is this?

- [x] Add a new dev container collection
- [ ] Update to an existing dev container collection
- [ ] Documentation/spec update
- [ ] Other containers.dev site update (UX, layout, etc)

## Description

Added Synology features. Currently, the only feature `synology-toolkit` facilitates development for Synology NAS devices utilizing the [Synology toolkit](https://help.synology.com/developer-guide/toolkit/toolkit.html)

### CODE REVIEW

File: _data/collection-index.yml [hunk 1]
```diff
@@ -822,4 +822,9 @@
   maintainer: Carsten Behring
   contact: https://github.com/scicloj/devcontainer-templates/issues
   repository: https://github.com/scicloj/devcontainer-templates/
-  ociReference: ghcr.io/scicloj/devcontainer-templates  
+  ociReference: ghcr.io/scicloj/devcontainer-templates
+- name: Synology devcontainer templates
```
Comment (samruddhikhandale) [inline _data/collection-index.yml]:
```suggestion
- name: Synology devcontainer Templates
```

Matching with Dev Containers terminology.

File: _data/collection-index.yml [hunk 2]
```diff
@@ -822,4 +822,9 @@
...
+  maintainer: ChaosWars
+  contact: https://github.com/ChaosWars/synology-features/issues
+  repository: https://github.com/ChaosWars/synology-features/
+  ociReference: ghcr.io/ChaosWars/synology-features
```
Comment (samruddhikhandale) [inline _data/collection-index.yml]:
```suggestion
  ociReference: ghcr.io/chaoswars/synology-features
```

Hi 👋

The ORAS spec requires the `ociReference` to be in lower case. Can we update here, thanks?

The supporting tools do help convert the case, however, its better to sync as much as possible.

Comment (samruddhikhandale) [inline _data/collection-index.yml]: Can you help change the visibility of this metadata package to public? The [index](https://containers.dev/templates) won't be able to crawl for the Templates in `chaoswars/synology-features` if it's private.

**Review by samruddhikhandale — CHANGES_REQUESTED** [review]

File: _data/collection-index.yml (hunk 2 above)
Comment (ChaosWars) [reply _data/collection-index.yml]: It should already be set to public, I just rechecked and it seems to be set correctly with visibility to public.

Comment (samruddhikhandale) [reply _data/collection-index.yml]: image

The toolkit package is correctly set to public, however, the metadata package doesn't seem to exist (it should have the name `synology-features`

Comment (ChaosWars) [reply _data/collection-index.yml]: My bad, `synology-toolkit/toolkiit`  was set to public, but `synology-features` was still set to private. It should be resolved now

**Review by samruddhikhandale — APPROVED** [review]
Thanks!

**Pull request merged**